other/upgrade_2-1_mysql.sql

```diff
@@ -963,6 +963,110 @@ ALTER TABLE {$db_prefix}members
 	DROP notify_announcements;
 ---#
 
+---# Creating alert prefs for watched topics
+---{
+	$_GET['a'] = isset($_GET['a']) ? (int) $_GET['a'] : 0;
+	$step_progress['name'] = 'Creating alert preferences for watched topics';
+	$step_progress['current'] = $_GET['a'];
+
+	$limit = 100000;
+	$is_done = false;
+
+	$request = $smcFunc['db_query']('', 'SELECT COUNT(*) FROM {db_prefix}log_notify WHERE id_member <> 0 AND id_topic <> 0');
+	list($maxTopics) = $smcFunc['db_fetch_row']($request);
+	$smcFunc['db_free_result']($request);
+
+	while (!$is_done)
+	{
+		nextSubStep($substep);
+		$inserts = array();
+
+		$request = $smcFunc['db_query']('', '
+			SELECT id_member, CONCAT(\'topic_notify_\', id_topic), 1
```

Comment:
i don't recommand to use concat, since it limit you to only two strings -> 
```
\'topic_notify_\' || id_topic
```
or to make it clearer
```
(\'topic_notify_\' || id_topic)
```
also maybe a good idea to add an alias to make refer
```
(\'topic_notify_\' || id_topic) as alert_pref
```

Reply:
Is that a psql thing? When I tried it on mysql it only returned 1 so I went for concat instead.
If it works for psql is there a point in using different methods for psql/mysql?

Reply:
i didn't know that this is in mysql default disable: https://github.com/SimpleMachines/SMF2.1/pull/6708
but than stay on concat, but still i would like to see that you provide an alias for this column.